test/conformance/testdata/conformance.yaml

```diff
@@ -1397,11 +1397,11 @@
 - testname: Kubectl, create service, replication controller
   codename: '[sig-cli] Kubectl client Kubectl expose should create services for rc  [Conformance]'
   description: Create a Pod running agnhost listening to port 6379. Using kubectl
-    expose the agnhost master replication controllers at port 1234. Validate that
+    expose the agnhost primary replication controllers at port 1234. Validate that
     the replication controller is listening on port 1234 and the target port is set
-    to 6379, port that agnhost master is listening. Using kubectl expose the agnhost
-    master as a service at port 2345. The service MUST be listening on port 2345 and
-    the target port is set to 6379, port that agnhost master is listening.
+    to 6379, port that agnhost primary is listening. Using kubectl expose the agnhost
+    primary as a service at port 2345. The service MUST be listening on port 2345 and
+    the target port is set to 6379, port that agnhost primary is listening.
```

Comment:
This was done by hand, there are other references to "master" in the conformance tests that are talking about:
```
$ kubectl cluster-info
Kubernetes master is running at https://127.0.0.1:42939
KubeDNS is running at https://127.0.0.1:42939/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy

To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
```

Which is a very different change. 

_some_ of those are really referring to the `kubernetes.default` / apiserver service as the "master" service (not `kubectl cluster-info` output) and could be changed trivially, but I think that's a different PR.